requirements.txt

```diff
@@ -4,83 +4,229 @@
-djangorestframework==3.12.1  # via -r requirements.in, djangorestframework-csv, drf-exceptions-hog, drf-extensions
-drf-exceptions-hog==0.0.4  # via -r requirements.in
-drf-extensions==0.6.0     # via -r requirements.in
-future==0.18.2            # via lzstring
-grpcio==1.33.1            # via -r requirements.in
-gunicorn==20.0.4          # via -r requirements.in
-idna==2.8                 # via -r requirements.in, requests
-importlib-metadata==1.6.0  # via -r requirements.in
-infi.clickhouse-orm==2.1.0  # via -r requirements.in
-iso8601==0.1.12           # via infi.clickhouse-orm
-kafka-helper==0.2         # via -r requirements.in
-kafka-python==2.0.1       # via -r requirements.in
-kombu==4.6.8              # via -r requirements.in, celery
-lxml==4.6.2               # via toronado
-lzstring==1.0.4           # via -r requirements.in
-monotonic==1.5            # via posthoganalytics
-numpy==1.18.1             # via -r requirements.in, pandas
-oauthlib==3.1.0           # via requests-oauthlib, social-auth-core
-pandas==1.1.3             # via -r requirements.in
-parso==0.6.1              # via -r requirements.in
-pexpect==4.7.0            # via -r requirements.in
-pickleshare==0.7.5        # via -r requirements.in
-posthoganalytics==1.1.3   # via -r requirements.in
-protobuf==3.13.0          # via -r requirements.in
-psycopg2-binary==2.8.4    # via -r requirements.in
-ptyprocess==0.6.0         # via pexpect
-pycparser==2.20           # via cffi
-pyjwt==1.7.1              # via social-auth-core
-python-dateutil==2.8.1    # via -r requirements.in, celery-redbeat, pandas, posthoganalytics
-python-statsd==2.1.0      # via -r requirements.in, django-statsd
-python3-openid==3.1.0     # via -r requirements.in, social-auth-core
-pytz==2019.3              # via -r requirements.in, celery, clickhouse-driver, django, infi.clickhouse-orm, pandas, tzlocal
-redis==3.4.1              # via -r requirements.in, celery-redbeat, django-redis
-requests-oauthlib==1.3.0  # via -r requirements.in, social-auth-core
-requests==2.22.0          # via -r requirements.in, django-rest-hooks, infi.clickhouse-orm, posthoganalytics, requests-oauthlib, social-auth-core
-sentry-sdk==0.19.2        # via -r requirements.in
-six==1.14.0               # via cryptography, djangorestframework-csv, grpcio, posthoganalytics, protobuf, python-dateutil, social-auth-app-django, social-auth-core, tenacity
-social-auth-app-django==3.1.0  # via -r requirements.in
-social-auth-core==3.3.3   # via -r requirements.in, social-auth-app-django
-sqlparse==0.3.0           # via django
-tenacity==6.1.0           # via celery-redbeat
-toronado==0.0.11          # via -r requirements.in
-tzlocal==2.1              # via clickhouse-driver
-unicodecsv==0.14.1        # via djangorestframework-csv
-urllib3==1.25.8           # via requests, sentry-sdk
-vine==1.3.0               # via amqp, celery
-whitenoise==5.0.1         # via -r requirements.in
-zipp==3.1.0               # via importlib-metadata
+aioch==0.0.2
```

Comment:
Interesting that the format changed. Makes it a bit harder to read

Reply:
Hmm, yeah, dunno why!